.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,54 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - ^v\d+\.\d+(\.\d+)?(-\S*)?$
+
+pr:
+- master
+
+resources:
+  repositories:
+  - repository: azure_template
+    type: github
+    name: mdolab/.github
+    endpoint: mdolab
+
+stages:
+- stage: Test
+  dependsOn: []
+  displayName: Test
+  jobs:
+  - job:
+    pool:
+      vmImage: 'ubuntu-20.04'
+    strategy:
+      matrix:
+        'python36':
+          PYTHON_VERSION: '3.6'
+        'python37':
+          PYTHON_VERSION: '3.7'
+        'python38':
+          PYTHON_VERSION: '3.8'
+    steps:
+    - script: sudo apt-get install texlive-pictures texlive-latex-extra -y
+      displayName: Prepare Repository
+    - script: |
+        python3 -m venv test
```

Comment:
What's the reason for using venv instead of the default system python? Was it because you couldn't get `testflo` working?

Reply:
Packages don't run natively with entry points in Azure, so you have to invoke them with `python3 -m package`. That does not work for testflo, so we switched to a virtual environment to house it to circumvent the issue.

Reply:
I believe the issue has to do with `$PATH`. Without Python properly set up, `$HOME/.local/bin` is not on the `$PATH` and that's why it can't find the program `testflo`. If it's still not on the PATH after setting up Python, you can manually add it and it should still work without venv.